CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.7.2 (2018-02-11)
+### Added
+  - New field: IMDb number of votes
```

Comment:
Add a reference at the end, to the Issue.
`- New field: IMDb number of votes #29`